charts/ingress-nginx/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: ingress-nginx
-version: 3.8.0
+version: 3.8.1
```

As this is a new feature please bump the minor version:
```suggestion
version: 3.9.0
```

---

Done